src/components/organisms/Slider/Slider.css

```diff
@@ -0,0 +1,135 @@
+:host {
+  --exposure: 50%;
+
+  --thumb-background-color: hsla(0, 0%, 100%, 0.9);
+  --thumb-background-image: url('data:image/svg+xml;utf8,<svg viewbox="0 0 60 60"  width="60" height="60" xmlns="http://www.w3.org/2000/svg"><path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="${thumbSvgWidth}" d="M20 20 L10 30 L20 40"/><path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="${thumbSvgWidth}" d="M40 20 L50 30 L40 40"/></svg>');
+  --thumb-size: clamp(3em, 10vmin, 5em);
+  --thumb-radius: 50%;
+  --thumb-border-color: hsla(0, 0%, 0%, 0.9);
+  --thumb-border-size: 2px;
+
+  --focus-width: var(--thumb-border-size);
+  --focus-color: hsl(200, 100%, 80%);
+
+  --divider-width: 2px;
+  --divider-color: hsla(0, 0%, 0%, 0.9);
+
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  overflow: hidden;
+  position: relative;
+}
+
+::slotted(img) {
+  height: auto;
+  width: 100%;
+}
+
+::slotted([slot='image-2']) {
+  clip-path: polygon(
+    calc(var(--exposure) + var(--divider-width) / 2) 0,
+    100% 0,
+    100% 100%,
+    calc(var(--exposure) + var(--divider-width) / 2) 100%
+  );
+}
+
+slot {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+slot[name='image-2'] {
+  position: absolute;
+  top: 0;
+  filter: drop-shadow(calc(var(--divider-width) * -1) 0 0 var(--divider-color));
+}
+
+.visually-hidden {
```

Comment:
@jedgar1mx bootstrap has a utility class for this already. 

https://getbootstrap.com/docs/5.3/helpers/visually-hidden/

Is the override of the bootstrap CSS intentional here?

Reply:
Nope, I just forgot to check bootstrap docs. Can you implement the bootstrap swap? Thanks

Reply:
Yep, no problem.